src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/NeedBracesCheck.java

```diff
@@ -72,8 +72,24 @@
  * <pre>
  * &lt;module name=&quot;NeedBraces&quot;&gt;
  *   &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_IF, LITERAL_ELSE&quot;/&gt;
+ *   &lt;property name=&quot;allowSingleLineStatement&quot; value=&quot;false&quot;/&gt;
```

Comment:
This is not the default property.
Default config is one without any property being explicitly set.
This is the default config: 
```xml
<module name="NeedBraces"/>
```
You have to look at the table and generate the exmples foro the same.